src/instructlab/model/evaluate.py

```diff
@@ -510,6 +510,16 @@ def evaluate(
     enable_serving_output,
 ):
     """Evaluates a trained model"""
+
+    # warn user if GPUs are not configured to be used
+    eval_serve = deepcopy(ctx.obj.config.serve)
+    is_gpus = gpus or eval_serve.vllm.gpus
+    if not is_gpus:
+        click.secho(
+            "Evaluate is currently not configured to use GPUs. If you are on a GPU-enabled system edit your config or pass the number of GPUs you would like to use with '--gpus'",
```

Comment:
Debating whether it's worth the check here for whether a gpu is available.

Reply:
I think that makes a lot of sense - no use warning the user if they don't have GPUs available - is there an existing func for this somewhere? Wasn't able to find anything

Reply:
It's a bit messy to cover all scenarios.  Check out what `ilab system info` is doing.